x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/bucket_span_estimator/estimate_bucket_span.ts

```diff
@@ -45,11 +45,10 @@ export function useEstimateBucketSpan() {
     indicesOptions: jobCreator.datafeedConfig.indices_options,
   };
 
-  if (
-    (isMultiMetricJobCreator(jobCreator) || isPopulationJobCreator(jobCreator)) &&
-    jobCreator.splitField !== null
-  ) {
+  if (isMultiMetricJobCreator(jobCreator) && jobCreator.splitField !== null) {
```

Comment:
I can't get the Estimate bucket span to work on rare jobs - using `gallery` here and `rare uri`:

![image](https://user-images.githubusercontent.com/7405507/122582826-7f802e00-d050-11eb-907c-f91080c69df3.png)

Reply:
This issue is not related to this PR. The bucket span estimator frequently fails me on `master` when creating a detector on the `gallery` data set when the partitioning field(s) gives series with lots of sparse data, for example when splitting on `clientip` or `uri`. I will create a separate issue for that.

Reply:
Created https://github.com/elastic/kibana/issues/103466 for separate investigation into the bucket span estimator.